tutorials/7-cosmjs/4-with-keplr.md

```diff
@@ -153,7 +153,7 @@ import { FaucetSender } from '../components/FaucetSender'
 const Home: NextPage = () => {
   return <FaucetSender
     faucetAddress="cosmos15aptdqmm7ddgtcrjvc5hs988rlrkze40l4q0he"
-    rpcUrl="https://rpc.sentry-01.theta-testnet.polypore.xyz" />
+    rpcUrl="rpc.sentry-01.theta-testnet.polypore.xyz" />
```

Comment:
Did this not work for you?

Reply:
Yes, with https I got CORS error, with http - 403 Forbidden, it worked only by removing it.